packages/iris-grid/src/GotoRow.tsx

```diff
@@ -28,6 +37,13 @@ interface GotoRowProps {
   onEntered: () => void;
   onExiting: () => void;
   onExited: () => void;
+
+  gotoValueSelectedColumn: Column;
+  gotoValueSelectedFilter: number;
```

Comment:
Would we also want a `containsIgnoreCase`? The API has case sensitivity and equals/contains separated. This is more of a question for Don, but would we be better off by including a separate toggle option that let's the user select if they want case sensitivity?

Reply:
Interestingly, the existing `FilterType.contains` is actually insensitive: https://github.com/deephaven/web-client-ui/blob/main/packages/jsapi-utils/src/TableUtils.ts#L1401. Removing the `(?i)` from that regex would make it case-sensitive.
We should probably also add `containsIgnoreCase` that uses regex `(?s)(?i).*\\Q${value}\\E.*`, then change `contains` to just use `(?s).*\\Q${value}\\E.*`. Not sure why it was omitted originally, will want to check with @dsmmcken when he's back.
It wouldn't require many changes, though it would change the behaviour of the `~` operator in quick filters: https://github.com/deephaven/web-client-ui/blob/main/packages/jsapi-utils/src/TableUtils.ts#L1335

Reply:
UI inputted values we always treat as case-insensitive.